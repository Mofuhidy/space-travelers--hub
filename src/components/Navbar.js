import logo from '../Assets/planet.png';

function Navbar() {
  return (
    <header>
      <div className="logo">
        <img src={logo} alt="logo" />
        <ul>
          <li>Rockets</li>
          <li>Missons</li>
          <li>My profile</li>
        </ul>
      </div>
    </header>
  );
}
export default Navbar;
