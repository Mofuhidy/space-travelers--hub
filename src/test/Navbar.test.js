import React from 'react';
import { render, screen } from '@testing-library/react';
import { BrowserRouter } from 'react-router-dom';
import Navbar from '../components/Navbar';

describe('Navbar', () => {
  it('Navbar navigation links work correctly', () => {
    render(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>,
    );

    const rocketsLink = screen.getByText('Rockets');
    expect(rocketsLink).toMatchSnapshot();
  });

  it('Navbar navigation links work correctly', () => {
    render(
      <BrowserRouter>
        <Navbar />
      </BrowserRouter>,
    );

    const myprofileLink = screen.getByText('My Profile');
    expect(myprofileLink).toMatchSnapshot();
  });
});
