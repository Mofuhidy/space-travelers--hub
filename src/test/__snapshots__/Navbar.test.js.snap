// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Navbar Navbar navigation links work correctly 1`] = `
<a
  aria-current="page"
  class="active"
  href="/"
>
  Rockets
</a>
`;

exports[`Navbar Navbar navigation links work correctly 2`] = `
<a
  class=""
  href="/myprofile"
>
  My Profile
</a>
`;
